datasets/superb/superb.py

```diff
@@ -213,8 +272,62 @@ class Superb(datasets.GeneratorBasedBuilder):
             url="https://github.com/ftshijt/LibriMix",
             data_url="https://huggingface.co/datasets/superb/superb-data/resolve/main/sd/{split}/{filename}",
         ),
+        SuperbConfig(
+            name="er",
+            description=textwrap.dedent(
+                """\
+            Emotion Recognition (ER) predicts an emotion class for each utterance. The most widely used ER dataset
+            IEMOCAP is adopted, and we follow the conventional evaluation protocol: we drop the unbalanced emotion
+            classes to leave the final four classes with a similar amount of data points and cross-validate on five
+            folds of the standard splits. The evaluation metric is accuracy (ACC)."""
+            ),
+            features=datasets.Features(
+                {
+                    "file": datasets.Value("string"),
+                    "label": datasets.ClassLabel(names=["neu", "hap", "ang", "sad"]),
+                }
+            ),
+            supervised_keys=("file", "label"),
+            url="https://sail.usc.edu/iemocap/",
+        ),
     ]
 
+    @property
+    def manual_download_instructions(self):
+        if self.config.name == "si":
+            return textwrap.dedent(
+                """
+            Please download the VoxCeleb dataset using the following script,
+            which should create `VoxCeleb1/wav/id*` directories for both train and test speakers`:
+            ```
+            mkdir VoxCeleb1
+            cd VoxCeleb1
+
+            wget https://thor.robots.ox.ac.uk/~vgg/data/voxceleb/vox1a/vox1_dev_wav_partaa
+            wget https://thor.robots.ox.ac.uk/~vgg/data/voxceleb/vox1a/vox1_dev_wav_partab
+            wget https://thor.robots.ox.ac.uk/~vgg/data/voxceleb/vox1a/vox1_dev_wav_partac
+            wget https://thor.robots.ox.ac.uk/~vgg/data/voxceleb/vox1a/vox1_dev_wav_partad
+            cat vox1_dev* > vox1_dev_wav.zip
+            unzip vox1_dev_wav.zip
+
+            wget https://thor.robots.ox.ac.uk/~vgg/data/voxceleb/vox1a/vox1_test_wav.zip
+            unzip vox1_test_wav.zip
+
+            # download the official SUPERB train-dev-test split
+            wget https://raw.githubusercontent.com/s3prl/s3prl/master/s3prl/downstream/voxceleb1/veri_test_class.txt
```

Comment:
windows users may not like this xD

Reply:
Can't wait to remove this if the VoxCeleb authors agree to move the files to the Hub haha